### PR TITLE
Feature/rds

### PR DESCRIPTION
**Pull request by DhruvJWork**
script for sqs trigger created

### CODE REVIEW

File: post-deployment-scripts/sqstrigger.sh [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+aws lambda create-event-source-mapping --function-name client-report-storage-master --batch-size 1 --maximum-batching-window-in-seconds 300 --event-source-arn arn:aws:sqs:eu-west-2:484165963982:dev-report-delivery-queue
```
Comment (jennifer-gft) [inline post-deployment-scripts/sqstrigger.sh]: event-source-arn must be passed as input variable to bash script, shouldn't h/c

Also report/add some success/failure response back from cli

File: post-deployment-scripts/populate.py [hunk 2]
```diff
@@ -13,12 +13,12 @@
                     datefmt='%H:%M:%S',
```
Comment (jennifer-gft) [inline post-deployment-scripts/populate.py:13]: Do not commit this to gh as these are sensitive info

File: post-deployment-scripts/logname [hunk 3]
```diff
@@ -0,0 +1,82 @@
+10:04:22,942 botocore.hooks DEBUG Changing event name from creating-client-class.iot-data to creating-client-class.iot-data-plane
```
Comment (jennifer-gft) [inline post-deployment-scripts/logname]: remove this file

**Review by jennifer-gft — CHANGES_REQUESTED** [review]

File: post-deployment-scripts/logname (hunk 3 above)
Comment (DhruvJWork) [reply post-deployment-scripts/logname]: resolved

File: post-deployment-scripts/populate.py (hunk 2 above)
Comment (DhruvJWork) [reply post-deployment-scripts/populate.py:13]: resolved

File: post-deployment-scripts/sqstrigger.sh (hunk 1 above)
Comment (DhruvJWork) [reply post-deployment-scripts/sqstrigger.sh]: resolved

**Review by jennifer-gft — APPROVED** [review]

**Pull request merged**